app/assets/javascripts/client/inventories/manage_inventory_access_requests.html

```diff
@@ -0,0 +1,38 @@
+<section class="permission-requests">
+  <h4>
+    Permission Requests <permissions-info data-placement="right"></permissions-info>
+  </h4>
+  <div ng-if="manageInventoryAccessRequests.list.length <= 0">
+    <h5>No Permission Requests</h5>
+  </div>
+
+  <div ng-if="manageInventoryAccessRequests.list.length > 0">
+    <div ng-repeat="requester in manageInventoryAccessRequests.list">
+
+      <div class="row row-participant-add">
+        <div class='col-sm-1'>
+          <img alt="" class="img-polaroid pull-left" ng-src="{{requester.avatar}}"/>
+        </div>
+        <div class='col-sm-2'>
+          <p class='name_label'>{{requester.full_name}}</p>
+        </div>
+        <div class='col-sm-3'>
+          <p class='info'>{{requester.email }}</p>
+        </div>
+        <div class='col-sm-3'>
+          <div class="pull-left permission_transition">
+            <span class="old_level">{{humanPermissionName(requester.current_permission_level.human)}}</span>
```

Comment:
Where's `humanPermissionName` defined at?  I'm not seeing it in the controller that's driving this view.